### ISSUE TITLE
Create folder when output folder does not exists in masking CLI

### ISSUE THREAD

**Issue by po09i** [shimming-toolbox/shimming-toolbox#238]
### Bug Description
When defining an output file when using `st_mask threshold ...` if the output folder does not exists, the call will crash. We should create the folder if it does not exist. We should check for the other masking CLIs as well.

### Reproduction Steps
Starting in the _shimming-toolbox/_ folder, make sure _temp/_ does not exist.

```
st_mask threshold -input testing_data/realtime_zshimming_data/nifti/sub-example/fmap/sub-example_magnitude1.nii.gz -thr 30 -output temp/mask.nii.gz
```

### Implementing the fix
The fix should be implemented in this file : shimmingtoolbox/cli/mask.py

**Comment by po09i** [comment]
@ahill187 I believe this was fixed in #248, if so we should add it to the fixes in the PR body.

**Comment by po09i** [comment]
This was fixed in the plugin's repo [here](https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/2). This can be closed.
